andela/ah-titans-frontend
# 160088425  Signup/Registration UI design with materialize

#### What does this PR do?
Design signup ui with the use of materialize.

#### Description of tasks to be completed
Implement designs for the Signup/Registration Page.

#### How should this be manually tested?
```sh
$ git clone https://github.com/andela/ah-titans-frontend.git
```
```sh
$ git checkout ft-revisit-signup-ui-design-160088425
```
```sh
$ cd ah-titans-frontend
```

>Install the dependencies using the command below.

```sh
$ npm install
```
>Run the application as below.
```sh
$ npm start
```
>Now you can navigate to your browser and you should be able to see a page as the one shown on the screenshot below.

![image](https://user-images.githubusercontent.com/29063646/44991911-5de85180-af9e-11e8-8879-aab58b5dbd44.png)




#### Any background context you want to provide?
No

#### What are the relevant Pivotal Tracker Stories?
[#160088425](https://www.pivotaltracker.com/story/show/160088425)
